src/networking.c

```diff
@@ -1349,13 +1349,43 @@ void acceptCommonHandler(connection *conn, struct ClientFlags flags, char *ip) {
         connClose(conn);
         return;
     }
+    in_addr_t ip_addr = 0;
+    if (sa != NULL && sa->ss_family == AF_INET) ip_addr = ((struct sockaddr_in *)sa)->sin_addr.s_addr;
+
+    if (server.trustedIPCount && ip_addr) {
+        if (/*!isTrustedNetwork(c) &&*/ !checkTrustedIP(ip_addr)) {
+            serverLog(LL_VERBOSE, "Access denied as connection is not from trusted source");
+
+            char *err = "-ERR client's IP is not found in trusted-addresses list, access denied\r\n";
+
+            /* That's a best effort error message, don't check write errors */
+            if (connWrite(conn, err, strlen(err)) == -1) {
+                /* Nothing to do, Just to avoid the warning... */
+            }
```

Comment:
Putting a (void) in front of a function call that returns value is a common way to say "I don't care about the return".
```suggestion
            (void)connWrite(conn, err, strlen(err));
```